poller.py

```diff
@@ -155,15 +166,15 @@ def _process(self, payload: dict) -> None:
 def main() -> None:
     args = parser.parse_args()
 
-    pipeline_config_path = find_config(args.config_path)
+    model_url = args.model_url
```

Comment:
There should be possibility to set `model_url` both in command line args and poller config file

Reply:
Done